### PR TITLE
Stage add window resize to page object

### PR DESCRIPTION
**Pull request by lujinappfolio**
Added support to get window dimension and resize windows.

### CODE REVIEW

File: ae_page_objects.gemspec [hunk 1]
```diff
@@ -4,8 +4,7 @@ require 'ae_page_objects/version'
 Gem::Specification.new do |s|
   s.name                      = "ae_page_objects"
   s.version                   = AePageObjects::VERSION
-
-  s.required_ruby_version     = '>= 1.9.3'
+  s.required_ruby_version     = '>= 2.0.0'
```
Comment (dtognazzini) [inline ae_page_objects.gemspec]: Please revert this change. Changing the required_ruby_version would necessitate a major version bump. Generally, we'd like to keep this gem available to most folks as possible.

File: lib/ae_page_objects/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module AePageObjects
-  VERSION = '2.0.0'.freeze
+  VERSION = '2.1.0.beta1'.freeze
```
Comment (dtognazzini) [inline lib/ae_page_objects/version.rb]: Please revert this change. The release process for this gem follows the common practice of releasing off of master.

**Comment by rmacklin** [comment]
From 1-1 conversation, this PR is no longer needed so the comments won't be addressed. Closing.
